frontend/src/index.jade

```diff
@@ -127,8 +129,9 @@ html
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title
                 .summary-chart__title--index {{ i+1 }}
-                .summary-chart__title--repo(v-if="!filterGroupRepos") {{ user.repoPath }}
-                .summary-chart__title--name {{ user.displayName }}
+                .summary-chart__title--repo(v-if="filterGroupSelection === 'groupByNone'") {{ user.repoPath }}
+                .summary-chart__title--author-repo(v-if="filterGroupSelection === 'groupByAuthors'") {{ user.repoPath }}
```

Comment:
is this just a difference in a styling? cos i dun see any stylesheet changes?

why need to have two `{{ user.repoPath }}`?

Reply:
`.summary-chart__title--repo` is bolded by default